src/cascadia/TerminalSettingsModel/Profile.h

```diff
@@ -117,7 +117,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         GETSET_SETTING(Microsoft::Terminal::TerminalControl::CursorStyle, CursorShape, Microsoft::Terminal::TerminalControl::CursorStyle::Bar);
         GETSET_SETTING(uint32_t, CursorHeight, DEFAULT_CURSOR_HEIGHT);
 
-        GETSET_SETTING(Model::BellStyle, BellStyle, BellStyle::Audible);
+        GETSET_SETTING(Model::BellStyle, BellStyle, BellStyle::All);

```

Comment:
Do we want visual bells to be the default? I'd think that the default would still be `audible`, but that's my opinion